test/sql/keywords/keywords_in_expressions.test

```diff
@@ -11,12 +11,18 @@ PRAGMA enable_verification;
 statement ok
 CREATE SCHEMA "SCHEMA";
 
-statement ok
+statement error
 CREATE TYPE "ENUM" AS ENUM('ALL');
```

Comment:
Is this change necessary?

Reply:
As a side effect it's now no longer possible to name your ENUM as ENUM. I could perhaps look into making this possible again if you like?

Reply:
Ah that’s alright, that makes sense - thanks for explaining